<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlignmentEditor</class>
 <widget class="QWidget" name="AlignmentEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>30</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>30</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="alignLeft">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../page_editor.qrc">
       <normaloff>:/images/format-justify-left.png</normaloff>:/images/format-justify-left.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="alignCenter">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../page_editor.qrc">
       <normaloff>:/images/format-justify-center.png</normaloff>:/images/format-justify-center.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="alignRight">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../page_editor.qrc">
       <normaloff>:/images/format-justify-right.png</normaloff>:/images/format-justify-right.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="alignJustify">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../page_editor.qrc">
       <normaloff>:/images/format-justify-fill.png</normaloff>:/images/format-justify-fill.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="alignTop">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../page_editor.qrc">
       <normaloff>:/images/format-justify-top.png</normaloff>:/images/format-justify-top.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="alignMiddle">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../page_editor.qrc">
       <normaloff>:/images/format-justify-middle.png</normaloff>:/images/format-justify-middle.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="alignBottom">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../page_editor.qrc">
       <normaloff>:/images/format-justify-bottom.png</normaloff>:/images/format-justify-bottom.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="wordWrap">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../page_editor.qrc">
       <normaloff>:/images/word-wrap.png</normaloff>:/images/word-wrap.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../page_editor.qrc"/>
 </resources>
 <connections/>
</ui>
